volume/types/volume.go

```diff
@@ -191,6 +191,11 @@ func (v *Volume) Key() string {
 	return v.Name
 }
 
+//CreateTime returns the volume's create time.
+func (v *Volume) CreateTime() string {
+	return v.CreationTimestamp.Format("2006-1-2 15:04:05")
```

Comment:
I found that in the definition we have:
```
	// CreationTimestamp is a timestamp representing the server time when this object was
	// created. It is not guaranteed to be set in happens-before order across separate operations.
	// Clients may not set this value. It is represented in RFC3339 form and is in UTC.
	CreationTimestamp *time.Time `json:"CreationTimestamp,omitempty"`
```
It seems to be a pointer, so I am wondering if we need to detect it is non-nil ?

Reply:
Added.